x-pack/plugins/maps/public/connected_components/map/mb/view.js

```diff
@@ -143,6 +147,14 @@ export class MBMap extends React.Component {
         mbMap.addControl(new mapboxgl.NavigationControl({ showCompass: false }), 'top-left');
       }
 
+      const hatchImageBase64 =
```

Comment:
Is this still being used. Looks like its now showing incomplete icon. If its not being used remove. If its being used, rename to something like tooManyFeaturesImage